src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -6,36 +6,35 @@ import { ROOT_DOMAIN_ID } from '@colony/colony-js';
 
```

Comment:
Please check this component very carefully (visually, especially), as this went through several tricky rebases